.github/workflows/ci.yaml

```
@@ -0,0 +1,95 @@
+          pytest tests/test_*.py -m "not integration"
+
+      - name: Run integration tests
+        run: |
+          source .venv/bin/activate
+          pytest tests/test_*.py -m integration
+        env:
+          NODE_REPO_DIR: ${{ github.workspace }}/tmp-node
+
+      - name: Cleanup
+        # not act 
+        if: always() && ${{ !env.ACT }}
```

actionlint (1.7.7)

if: condition "always() && ${{ !env.ACT }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)